### ISSUE TITLE
nufft calls lambertw, which is only available in the symbolic toolbox

### ISSUE THREAD

**Issue by nickhale** [chebfun/chebfun#2165]
```
>> which lambertw
/usr/local/MATLAB/R2017a/toolbox/symbolic/symbolic/@single/lambertw.m  % single method
```

**Comment by nickhale** [comment]
As does `nufft2`.

I recommend using a lookup in Table 2.1 from @ajt60gaibb 's paper instead.

**Comment by trefethen** [comment]
This needs to be fixed for v5.7.0.
